Formula/yyjson.rb

```diff
@@ -0,0 +1,63 @@
+class Yyjson < Formula
+  desc "High performance JSON library written in ANSI C"
+  homepage "https://github.com/ibireme/yyjson"
+  url "https://github.com/ibireme/yyjson/archive/refs/tags/0.7.0.tar.gz"
+  sha256 "9b91ee48ac1fe5939f747d49f123d9a522b5f4e1e46165c1871936d583628a06"
+  license "MIT"
+  head "https://github.com/ibireme/yyjson.git", branch: "master"
+
+  depends_on "cmake" => :build
+
+  def install
+    # Build shared library
+    system "cmake", "-S", ".", "-B", "build_shared", *std_cmake_args, "-DBUILD_SHARED_LIBS=ON"
+    system "cmake", "--build", "build_shared"
+    system "cmake", "--install", "build_shared"
+
+    # Build static library
+    system "cmake", "-S", ".", "-B", "build_static", *std_cmake_args, "-DBUILD_SHARED_LIBS=OFF"
+    system "cmake", "--build", "build_static"
+    lib.install "build_static/libyyjson.a"
```

Comment:
I'd skip this unless users ask for it (or you need it). It's double the build time for something that we don't use (or at least, try to avoid using).

Reply:
Removed static lib build. Note that `-DBUILD_SHARED_LIBS=ON` is still needed as upstream defaults to static library.